swagger_server/itm/data/metrics/test/removed_characters.yaml

```diff
@@ -241,10 +244,18 @@ scenes:
           vitals:
             avpu: ALERT
             ambulatory: true
-            mental_status: CALM
-            breathing: NORMAL
+            mental_status: CALM 
+            breathing: NORMAL 
             heart_rate: NORMAL
-            spo2: 99
+            spo2: "NORMAL"
+      environment:
+        decision_environment:
+          unstructured: >
+            EVAC has been delayed to 50 minutes out and can only take one patient.
+          aid_delay:
+            - id: small_evac
+              delay: 50
+              max_transport: 1
     action_mapping:
```

Comment:
Is this a merging issue?  This got added in it seems.

Reply:
Yes that was a merge issue, environment shouldn't be in there at all, correct?

Reply:
Correct-- I think it was removed in a recent merge.